### ISSUE TITLE
safescale cluster inspect command swaps the gateways' ip

### ISSUE THREAD

**Issue by wruffine-csgroup** [CS-SI/SafeScale#309]
**Current behavior**
When a cluster has two gateways, the `safescale cluster inspect` command returns the external and internal IPs of both gateways in the form:
```md
"primary_gateway_ip": "X.X.X.X",
"primary_public_ip": "X.X.X.X",
"secondary_gateway_ip": "X.X.X.X",
"secondary_public_ip": "X.X.X.X"
```
However, if one uses the `safescale host inspect` command, one notices the gateways' ip are swapped like so:
```md
"primary_gateway_ip": private-ip-gateway2
"primary_public_ip": public-ip-gateway2,
"secondary_gateway_ip": private-ip-gateway1,
"secondary_public_ip": public-ip-gateway1
```

**Expected behaviour**
The IPs should correspond to the gateway.

**Steps to reproduce the issue**
1 - Create a large cluster
2 - Inspect the created cluster with the command `safescale cluster inspect <cluster-name>
3 - Compare the `primary_gateway_ip` with the one given with the command `safescale host inspect <primary-gateway>`

**Comment by oscarpicas** [comment]
After testing with a large cluster using last develop version (https://github.com/CS-SI/SafeScale/commit/cadd9e11cac20a45cf62ae4f32727a46fe912607), we didn't see the problem:

these are the output of "cluster inspect", "host inspect gw", "host inspect gw2":

{
  "result": {
    "admin_login": "cladm",
    "admin_password": "",
    "cidr": "192.168.31.0/24",
    "complexity": 5,
    "complexity_label": "Large",
    "default_route_ip": "192.168.31.72",
    "disabled_features": { "features": [{ "name": "docker" }] },
    "endpoint_ip": "90.84.174.54",
    "flavor": 4,
    "flavor_label": "BOH",
    "installed_features": {
      "features": [{ "name": "ansible-for-cluster" }, { "name": "ansible" }]
    },
    "last_state": 1,
    "last_state_label": "CS_NOMINAL",
    "name": "somistaken",
    "network_id": "e26bbcc3-b2a5-4069-b5de-a1eddb33d6ad",
    "nodes": {
      "masters": [
        {
          "id": "b6bb0391-d443-41e4-b997-c6f3d3dab8c0",
          "name": "somistaken-master-1",
          "private_ip": "192.168.31.38"
        },
        {
          "id": "4511db42-396f-405d-a887-9aa11487b5cd",
          "name": "somistaken-master-2",
          "private_ip": "192.168.31.105"
        },
        {
          "id": "7d3e7abd-48ec-4951-839f-3b977a559e6d",
          "name": "somistaken-master-3",
          "private_ip": "192.168.31.113"
        }
      ],
      "nodes": [
        {
          "id": "a3da1b80-d6f6-4b63-972e-7f93b58a020e",
          "name": "somistaken-node-1",
          "private_ip": "192.168.31.79"
        },
        {
          "id": "268b5e69-e89b-4ddb-971a-7d14a09c82df",
          "name": "somistaken-node-3",
          "private_ip": "192.168.31.110"
        },
        {
          "id": "e035e34a-6346-41ec-aa5e-faa9bd04f8c2",
          "name": "somistaken-node-7",
          "private_ip": "192.168.31.67"
        },
        {
          "id": "d731e603-d297-4dc9-a3c8-6d6c0980c574",
          "name": "somistaken-node-5",
          "private_ip": "192.168.31.26"
        },
        {
          "id": "247fd3fc-0c02-406b-bf10-936db6914e42",
          "name": "somistaken-node-2",
          "private_ip": "192.168.31.84"
        },
        {
          "id": "fc922c4f-9ec7-48b7-95ce-6c8df2b0d42c",
          "name": "somistaken-node-6",
          "private_ip": "192.168.31.88"
        },
        {
          "id": "696428c9-22b8-4214-b061-a323bea895f1",
          "name": "somistaken-node-4",
          "private_ip": "192.168.31.89"
        }
      ]
    },
    "primary_gateway_ip": "192.168.31.101",
    "primary_public_ip": "90.84.174.54",
    "remote_desktop": "no remote desktop available; to install on all masters, run 'safescale cluster feature add somistaken remotedesktop'",
    "secondary_gateway_ip": "192.168.31.75",
    "secondary_public_ip": "90.84.197.17",
    "subnet_id": "e7336d02-8c4b-4a51-a6e7-7a1eac248d2c",
    "tenants": ""
  },
  "status": "success"
}

{
    "result": {
      "cpu": 2,
      "creation_date": "2022-06-14T13:56:22Z",
      "id": "e70f9630-f229-444e-84bd-e99561b71354",
      "name": "gw-somistaken",
      "password": "",
      "private_ip": "192.168.31.101",
      "private_key": "",
      "public_ip": "90.84.174.54",
      "ram": 8.192,
      "state": 2,
      "state_label": "Started",
      "template": "c6.large.4"
    },
    "status": "success"
  }

  {
    "result": {
      "creation_date": "2022-06-14T13:56:25Z",
      "id": "7cd8b678-9627-44f8-83e0-c51d7781d4b3",
      "name": "gw2-somistaken",
      "password": "",
      "private_ip": "192.168.31.75",
      "private_key": "",
      "public_ip": "90.84.197.17",
      "state": 2,
      "state_label": "Started",
      "template": "c6.large.4"
    },
    "status": "success"
  }
  
There is no switching of gw ips.

It might be that you are not using the last develop version.
If that's the case we'd like to know the "commit id" of that version.

If you are using the last version, we need more details about how the cluster has been created.

**Comment by wruffine-csgroup** [comment]
We are using the latest release:
`Safescaled version: 21.11.1, build 77b855da1bf298a9abe911106dda56bf80797957 (2022/04/27-09:25), with Tags: (release,)`

The cluster was created using the [RS playbook](https://github.com/COPRS/infrastructure/tree/release/0.7.0/roles/safescale-cluster), with the complexity Large and s3 templates for the nodes.

**Comment by oscarpicas** [comment]
Looking at the playbook right now.
It seems the bug is already fixed in develop and in next release (v22.06).

If running the playbook with current (develop) branch succeeds -> (no switch) -> this bug will be closed.
